generators/heroku/templates/application-heroku.yml.ejs

```diff
@@ -65,6 +65,12 @@ jhipster:
     <%_ } _%>
 <%_ } _%>
 spring:
+<%_ if (serviceDiscoveryType === 'eureka') { _%>
+    cloud:
+        config:
+            fail-fast: true
+            uri: ${JHIPSTER_REGISTRY_URL}/config
```

Comment:
The Heroku test is failing because of the YML indentation.  Got the error message by running `jhipster heroku` in a folder with the `default-microservice` .yo-rc.json and providing the same prompts as the test.
```suggestion
  cloud:
    config:
      fail-fast: true
      uri: ${JHIPSTER_REGISTRY_URL}/config
```

Reply:
Good eye, @ruddell! YAML indents are my nemesis. 🤬